src/Components/Movies/singleMovie.js

```diff
@@ -1,17 +1,29 @@
 import React from 'react'
+import PropTypes from 'prop-types'
 
-
-const SingleMovie = ({filteredMovie, returnHome}) => {
-  return (
-    <div className='single-movie' >
-      <img className='single-poster' src={filteredMovie[0].backdrop_path} ></img>
-      <div className='text-details'>
-        <p>{filteredMovie[0].title}</p>
-        <p>{filteredMovie[0].average_rating}</p>
-        <button onClick={() => returnHome()} >Return</button>
+const SingleMovie = ({filteredMovie, returnHome}) =>  {
+  console.log('filtered', filteredMovie)
+    return (
+      <div className='single-movie' >
+        <img className='single-poster' src={filteredMovie.backdrop_path} ></img>
+        <div className='text-details'>
+          <p>{filteredMovie.title}</p>
+          <p>{filteredMovie.average_rating}</p>
+          <p>{filteredMovie.budget}</p>
+          <p>{filteredMovie.overview}</p>
+          <p>{filteredMovie.runtime}</p>
+          <p>{filteredMovie.release_date}</p>
```

Comment:
This could be a good opportunity to add some destructuring so you don't have to type `filteredMovie.____` so many times.  Not required though.